template.tpl

```diff
@@ -1268,6 +1355,10 @@ const generateConfiguration = () => {
     initOptions.defaultTracking = false;
   }
 
+  if(initOptions.logLevel == 4){
+    log(LOG_PREFIX + 'INFO: ' + "Amplitude instance will be initialized by configuration: " + JSON.stringify(initOptions));
```

Comment:
It is good to add more logs. However, we require the JSON stringify only for logging purposes, I feel that's too much. Not a blocker though.